.github/workflows/cluster_endtoend_11.yml

```diff
@@ -1,3 +1,5 @@
+# DO NOT MODIFY: THIS FILE IS GENERATED USING test/ci_workflow_gen.go
```

Comment:
What do you think about saying that these files are generated using `make generate_ci_workflows`?

Reply:
Good idea, will make the change